tests/e2e/tests/regression/cli/cli-promote-workbench.ts

```diff
@@ -21,16 +21,16 @@ fixture `Promote workbench in CLI`
         // Delete database
         await deleteStandaloneDatabaseApi(ossStandaloneConfig);
     });
-test('Verify that users can see workbench promotion message when they open CLI', async t => {
-    // Open CLI
-    await t.click(cliPage.cliExpandButton);
-    await t.expect(cliPage.workbenchLink.parent().textContent).eql('Try Workbench, our advanced CLI. Check out our Quick Guides to learn more about Redis capabilities.');
-    // Verify that user is redirected to Workbench page clicking on workbench link in CLI
-    await t.click(cliPage.workbenchLink);
-    await t.expect(workbenchPage.expandArea.exists).ok('Workbench page is opened');
-    // Verify that CLI panel is minimized after redirection to workbench from CLI
-    await t.expect(cliPage.cliPanel.visible).notOk('Closed CLI');
-});
+// test('Verify that users can see workbench promotion message when they open CLI', async t => {
```

Comment:
?

Reply:
thanks for noticing, I forgot to delete some already merged tests